lib/ansible/module_utils/network/nxos/nxos.py

```diff
@@ -739,13 +739,16 @@ class NxosCmdRef:
           multiplier: 3
     """
 
-    def __init__(self, module, cmd_ref_str):
+    def __init__(self, module, cmd_ref_str, ref_only=None):
```

Comment:
I'd prefer using optional keyword args, not a fan of positional options and I have a feeling this module will grow its options before long.
e.g.
```
    def __init__(self, module, cmd_ref_str, **kwargs):
        ...
        ref_only = kwargs.get('ref_only', False)
```

Callers can then:
```
    NxosCmdRef(self._module, TMS_GLOBAL, ref_only=True)
```